frontend/src/static/js/v_authorship.js

```diff
@@ -92,10 +92,11 @@ window.vAuthorship = {
     },
 
     setInfoHash() {
-      const { addHash, removeHash } = window;
+      const { addHash, removeHash, encodeHash } = window;
       addHash('tabAuthor', this.info.author);
       addHash('tabRepo', this.info.repo);
       removeHash('tabOpen');
+      encodeHash();
```

Comment:
what does this has to do with encodeHash though?

Reply:
Previously when you open the authorship tab, the watcher `repos` will be triggered and the `getFiltered()` method will be called. The `getFiltered()` method calls `setSummaryHash` which calls the `encodeHash()` method.

Now since we removed the watcher for `repos`, `getFiltered` won't be called when you open the authorship tab, the `tabAuthor` and `tabRepo` hash will not be reflected in the URL. That is why I added the encodeHash here that adds `tabRepo` and `tabAuthor` when the `tabAuthorship` component is created.